feat(content): Weland The Xeno-Smith - Human Mission (#10565)
endless-sky/endless-sky: data/human/human missions.txt

	destination "Wayfarer"
	to offer
		has "event: welandwait2"
		not "Dragon Tech 1b: offered"
		not "Dragon Tech 1c: offered"
		not "Dragon Tech 1d: offered"
	on offer
		require "Blood of Surtr" 0
		require "Tear of Skade" 0
		dialog
			`Your inbox blinks with a new message from Weland on Wayfarer.`
			`	<name>, this is Weland. Just letting you know your artifact is ready.`
			`	Come pick it up and pay for it at your earliest convenience.`
			`				-Xeno-Smith`
			`					Weland`
			`	P.S. Make sure you have 5 tons of outfit space!`
	on complete
		event "welandstone" 66
		conversation
			`You find Weland inside his makeshift laboratory. Your artifact, covered in cords and various machinery, is no longer dull and inert, but has whirred to life.`
			`	"Take a moment. It's a rare sight, alright. Watchin' somethin's been dormant for longer than mankind's had words spur to life before ya, why, it's enough to give a man religion, or something near abouts." Weland pats you on the back.`
			`	"Now, what we have here is an Fusion Afterburner, it's an afterburner. From what I can tell, despite it's age, a pretty effective one too."`

> 			`	Weland taps a datapad on his workbench. "And in here is everything you need to install it on your ship."`

Reviewer: ```suggestion
			`	Weland taps a datapad on his workbench. "And in here is everything you need to install it on your ship.`
```

	source
		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
	destination "Wayfarer"
	to offer
		has "event: welandwait2"
		not "Dragon Tech 1: offered"
		not "Dragon Tech 1c: offered"
		not "Dragon Tech 1d: offered"
	on offer
		require "Blood of Surtr"
		require "Tear of Skade" 0
		dialog
			`Your inbox blinks with a new message from Weland on Wayfarer.`
			`	<name>, this is Weland. Just letting you know your artifact is ready.`
			`	Come pick it up, and pay for it, at your earliest convenience.`
			`				-Xeno-Smith`
			`					Weland`
			`	P.S. Make sure you have 5 tons of outfit space!`
	on complete
		conversation
			`You find Weland inside his makeshift laboratory. Your artifact, covered in cords and various machinery, is no longer dull and inert, but has whirred to life.`
			`	"Take a moment. It's a rare sight, alright. Watchin' somethin's been dormant for longer than mankind's had words spur to life before ya, why, it's enough to give a man religion, or something near abouts." Weland pats you on the back.`
			`	"Now, what we have here is an Fusion Afterburner, it's an afterburner. From what I can tell, despite it's age, a pretty effective one too."`

> 			`	Weland taps a datapad on his workbench. "And in here is everything you need to install it on your ship."`

Reviewer: ```suggestion
			`	Weland taps a datapad on his workbench. "And in here is everything you need to install it on your ship.`
```

			`	"I'll miss having it around. Made the place feel mysterious." He shrugs.`
			action
				outfit "Retrofitted Fusion Afterburner" 1
			`	"Now that we got that all settled. There's one other item I was wantin' to discuss. Somethin' in the ship schematics you sent over caught my eye. I think you had it labeled, 'Tear of Skade'. Now, if that is what I think it is, well, it's worth a lot of money. I never met nobody that has laid eyes on a Deep Stone, let alone owned one. It's phantom tech kind of stuff you hear about from spooks, and conspiracy nuts. Most say it isn't even real, and if it is, only a few ever been found, and those the government holds under lock and key.`
			`	"But, here you are flying free with one on your ship doin' whatever it is these strange stones do. I'd as pay a pretty penny for somethin' like that. How about I waive the fee for fixin' up your artifact, and pay you 10 million credits for that 'Tear of Skade' stone of yours?"`
			choice
				`	(Refuse to sell the Tear of Skade.)`
					goto refuse
				`	(Agree to sell the Tear of Skade.)`

> 			`	Weland's grins like hyena, "Hot damn. Know that this is just about every red cent I own, and in some cases don't. But with this stone, and the secrets it holds, who knows what I'll discover."`

Reviewer: ```suggestion
			`	Weland grins like a hyena, "Hot damn. Know that this is just about every red cent I own, and in some cases don't. But with this stone, and the secrets it holds, who knows what I'll discover."`
```

			action
				outfit "Tear of Skade" -1
				event "welandwait3skade" 90

event "welandwait3skade"

mission "Dragon Tech 1d"
	name "Return to Weland on Wayfarer"
	description "Weland is done working on your artifact, and has requested that you return to Wayfarer and meet him."
	source
		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
	destination "Wayfarer"
	to offer
		has "event: welandwait2"
		not "Dragon Tech 1b: offered"
		not "Dragon Tech 1c: offered"
		not "Dragon Tech 1: offered"
	on offer
		require "Blood of Surtr"
		require "Tear of Skade"
		dialog
			`Your inbox blinks with a new message from Weland on Wayfarer.`
			`	<name>, this is Weland. Just letting you know your artifact is ready.`
			`	Come pick it up, and pay for it, at your earliest convenience.`
			`				-Xeno-Smith`
			`					Weland`
			`	P.S. Make sure you have 5 tons of outfit space!`
	on complete
		conversation
			`You find Weland inside his makeshift laboratory. Your artifact, covered in cords and various machinery, is no longer dull and inert, but has whirred to life.`

> 			`	"Take a moment. It's a rare sight, alright. Watchin' somethin's been dormant for longer than mankind's had words spur t#o life before ya, why, it's enough to give a man religion, or something near abouts." Weland pats you on the back.`

Reviewer: ```suggestion
			`	"Take a moment. It's a rare sight, alright. Watchin' somethin's been dormant for longer than mankind's had words spur to life before ya, why, it's enough to give a man religion, or something near abouts." Weland pats you on the back.`
```
Oops.

			`	"But, here you are, flying free with not one, but two, on your ship, doin' whatever it is these strange stones do. I'd as pay a pretty penny for somethin' like that. How about I waive the fee for fixin' up your artifact, and pay you 10 million credits for one of those stones of yours?"`
			choice
				`	(Refuse to sell either stone.)`
					goto refuse
				`	(Agree to sell the Tear of Skade.)`
					goto soldskade
				`	(Agree to sell the Blood of Surtr.)`
			`	Weland grins like hyena, "Hot damn. Know, that this is just about every red cent I own, and in some cases don't. But with this stone, the secrets it holds, who knows what I'll discover."`
			`	He laughs, maniacally, and transfers the credits over to you.`
			action
				payment 10000000
				outfit "Blood of Surtr" -1
			`	He hobbles over to you, and shakes your hand. "Pleasure doin' business. And perhaps we'll run into each other again."`
				decline
			label soldskade

> 			`	Weland's grins like hyena, "Hot damn. Know, that this is just about every red cent I own, and in some cases don't. But with this stone, the secrets it holds, who knows what I'll discover."`

Reviewer: ```suggestion
			`	Weland grins like a hyena, "Hot damn. Know, that this is just about every red cent I own, and in some cases don't. But with this stone, the secrets it holds, who knows what I'll discover."`
```

		conversation
			`You find Weland tinkering away in his shop. He puts his inscrutable tools down and waves to you as you enter.`
			`	"Ah, there you are. I am sure you're wonderin' yourself sick trying to divine why I called ya." He grins, and limps toward you, dragging metal chains behind his feet.`
			`	"I heard word, from someone who's done crew work on your ship, that you're in possession of a couple of Deep Stones." He nearly whispers the last the two words.`
			`	"I've been told you call 'em, the 'Tear of Skade', and the 'Blood of Surtr'". He watches your eyes carefully as he names off the stones. "Now, if they are what I think they are, well, they're worth a lot of money. I never met nobody that has laid eyes on a Deep Stone, let alone owned one. It's phantom tech, kind of stuff you hear about from spooks, and conspiracy nuts. Most say it isn't even real, and if it is, only a few ever been found, and those the government holds under lock and key.`
			`	"But, here you are, flying free with not one, but two, on your ship, doin' whatever it is these strange stones do. I'd offer to buy one of 'em off ya, but I don't have anywhere nears enough cash for one of those.`

> 			`	"So, what if I borrowed one of the stones? Just, hear me out, alright. Give me three months to study it, learn what secrets from it I can. I'd pay you 3 million credits for the pleasure. After 3 months, you come collect the stone, and your payment. Everyone wins. Whatd'ya say?"`

Reviewer: The way this is written, it sounds like there was a possibility not everybody would be happy.
Maybe remove "Everyone wins" all together.

Author: It's a sales pitch, though. He's trying to convince you to do it.

				outfit "Tear of Skade" -1
			`	Weland grins. "A little time's better than no time. Put 'er there."`
			`	He takes your hand and shakes it with great enthusiasm. "Imagine, what secrets await."`
			`	Later on you transfer over the stone, 'Tear of Skade', to a feverishly eager Weland. You promise to back in three months, for stone and payment.`
				decline
			label studysurtrweland
			`	Weland grins. "A little time's better than no time. Put 'er there."`
			`	He takes your hand and shakes it with great enthusiasm. "Imagine, what secrets await."`
			`	Later on you transfer over the stone, 'Blood of Surtr', to a feverishly eager Weland. You promise to back in three months, for stone and payment.`
			action
				event "welandwait3surtr" 2
				outfit "Blood of Surtr" -1

mission "Stone Harmonization 2skade"
	name "Return to Weland on Wayfarer"
	description "Weland has a mysterious problem and wants you to meet him on Wayfarer as soon as possible."
	source
		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
	destination "Wayfarer"
	to offer
		has "event: welandwait3skade"
	on offer
		dialog
			`Your inbox blinks with a new message from Weland on Wayfarer.`

> 			`	'<name>, we have a problem. Get back here ASAP.`

Reviewer: ```suggestion
			`	<name>, we have a problem. Get back here ASAP.`
```

		"LP144a Battery Pack"
		"D41-HY Shield Generator"
		"Large Radar Jammer"
		"Liquid Nitrogen Cooler"
		"Laser Rifle" 15

		"Impala Plasma Thruster"
		"Impala Plasma Steering"
		"Hyperdrive"

ship "Falcon" "Falcon (Corentin Variant)"
	outfits
		"Blood of Surtr"
		"Plasma Cannon" 2
		"Torpedo Launcher" 2
		"Torpedo" 60
		"Quad Blaster Turret" 3
		"Heavy Anti-Missile Turret"

		"Breeder Reactor"
		"KP-6 Photovoltaic Array"
		"KP-6 Photovoltaic Panel" 2
		"LP144a Battery Pack"
		"D41-HY Shield Generator"
		"Large Radar Jammer"
		"Liquid Nitrogen Cooler"
		"Laser Rifle" 15

		"Impala Plasma Thruster"
		"Impala Plasma Steering"
		"Hyperdrive"

mission "Stone Harmonization 3skade"
	name "Board the Aotrou"
	description "Find the Aotrou, disable it, board it, and retrieve the Tear of Skade."
	source "Wayfarer"
	waypoint "Naper"
	destination "Wayfarer"
	to offer
		has "Stone Harmonization 2skade: done"
	on offer
		conversation
			`Weland is waiting for you outside a nearly empty eatery. He turns his wheelchair to face you as you approach.`
			`	"Well, as you may have surmised," he nods toward his covered feet, that were until recently, chained, "I am someone else's property."`

> 			`	The injured man sighs, and looks away from you, "They let me do my own work, follow my own interests, pretend to be my own man, so long as I stay put, and do whatever they ask, when they ask it."`

Reviewer: ```suggestion
			`	The injured man sighs, and looks away from you, "They let me do my own work, follow my own interests, pretend to be my own man, so long as I stay put, and do whatever they ask, when they ask it.`
```

			choice
				`	"I'll do it."`
					goto doit
				`	"I'll take back what is mine, and I'll pay them back for what they did to you."`
					goto doit
				`	"Not interested."`
					goto no
				`	"You made the promise, you owe me, not them. Make me whole."`
			`	Weland frowns. "Thought you a little less common than that. Well, my mistake."`
			`	He pulls out a keypad, and hits a few buttons. "I wager about 10 million will make you whole. If it doesn't, well it's all I got, alright. I've transferred over the money. Take, and get out of my sight. I don't deal with common."`
			action
				payment 10000000
			`	Weland wheels away from you, leaving you alone on the muddy street.`
				decline
			label no
			`	Weland frowns. "Thought you a little less common than that. Well, my mistake."`
			`	He pulls out a keypad, and hits a few buttons. "I can't get your stone back, but least I can do is pay you the 3 million I promised. I'm right sorry there isn't more I can do, but, they still own me, and there is only so much I can do."`
			action
				payment 3000000
			`	Weland nods at you, and then wheels away, back toward his shop, leaving you alone on the muddy street.`
				decline
			label doit

> 			`	Weland grins. "I never took you for a lily livered captain of a paper ship, glad to see I was right."`

Reviewer: ```suggestion
			`	Weland grins. "I never took you for a lily livered captain of a paper ship, glad to see I was right.`
```

			`	"The ship's called the 'Aotrou', and it's captained by Corentin Gwesklin, of the Gwesklin Slaver-Dynasty. That family tricked me, and has held me in bondage longer than most folk's memories; though there is little enough you can do 'bout that now."`

> 			`	Weland opens up a palm-display with star coordinates on it. "They are part of the Core Mining Concern. Semi-Sanctioned criminals, slavers, and all around bad guys. Led by the likes of the Gwesklin, and other hereditary slaver families. Though they can travel through civilized space unmolested, they do like to lay low in the Naper system, especially after altercations. That's where I'd wager you'd find Mr. Gwesklin, his ship, and the stone he stole from us.`

Reviewer: ```suggestion
			`	Weland opens up a palm-display with star coordinates on it. "They are part of the Core Mining Concern. Semi-sanctioned criminals, slavers, and all around bad guys. Led by the likes of the Gwesklin, and other hereditary slaver families. Though they can travel through civilized space unmolested, they do like to lay low in the Naper system, especially after altercations. That's where I'd wager you'd find Mr. Gwesklin, his ship, and the stone he stole from us.`
```

			`	"Be warned, he most likely will scan as a friendly, the C.M.C. leadership don't find it profitable to run pirate transponders. So keep your eyes open. Now, I think that's 'bout all needs being said at this juncture, just be sure to come back here once you got the stone. I got a surprise for ya."`
			`	Weland leans forward, and pats you on the arm, before turning his wheelchair around, and heading back in the direction of his shop. After a moment you're alone on a muddy street corner.`
				accept

	npc board
		government "Bounty (Disguised)"
		system "Naper"
		personality entering target timid staying uninterested
		ship "Falcon" "Falcon (Gwesklin Variant)" "C.M.C.S. Aotrou"

> 		dialog `It looks like Captain Gwesklin took an escape pod and abandoned the ship earlier in the battle. The stone is still here, though.`

Reviewer: ```suggestion
		dialog `It looks like Captain Gwesklin took an escape pod and abandoned the ship earlier in the battle. The stone is still here, though. You should take it.`
```

mission "Stone Harmonization 4skade"
	source "Wayfarer"
	invisible
	to offer
		has "Stone Harmonization 3skade: done"
	on offer
		require "Tear of Skade"
		payment 3000000
		outfit "Tear of Skade" -1
		outfit "Tear of Skade (Harmonized)" 1
		conversation
			`Weland's laboratory has mostly been cleaned up, and repaired. Weland is waiting for you at his work bench, and gently places the Tear of Skade on some sort of device.`
			`	"It sings." Weland almost whispers, as a display shows acoustic waves emanating from within the stone. "The Crystalline lattice within has been re-arranged in ways no one has ever dreamt of, in order to generate and produce waves that appear to alter the state of matter."`
			`	Weland shakes his head, "Not only is this far advanced beyond anything humans could create, or recreate, but it's an entire untapped field of study." Weland almost trembles with awe, "It's remarkable. Neither we, nor humans, nor any other race I know of, has deigned to interact with rock, crystal, and matter in this fashion."`
			choice

> 				`	"What do you mean, "we", aren't you human?"`

Reviewer: ```suggestion
				`	"What do you mean, 'we'? Aren't you human?"`
```

			`	He shrugs, "Perhaps, some other time. On a different world, in different circumstances. But, not now, not here, and not like this."`
				goto questions
			label offworld
			`	He gives you a patronizing smile. "Folks that did this to me. They've been doin' this kind of work for a while, and I, well, I am like the crown jewel of their collection. They have hooks into me, inside of me, all around me, that prevent me from leaving, at least alive. They like to give me the illusion of space, pretend they are... gentile 'bout the whole thing. But make no mistake, I am not free, and I cannot go."`
			`	His face briefly flashes with mirth, "For now, that is."`
				goto questions

mission "Stone Harmonization 3surtr"
	name "Board the Aotrou"
	description "Find the Aotrou, disable it, board it, and retrieve the Blood of Surtr."
	source "Wayfarer"
	waypoint "Naper"
	destination "Wayfarer"
	to offer
		has "Stone Harmonization 2surtr: done"
	on offer
		conversation
			`Weland is waiting for you outside a nearly empty eatery. He turns his wheelchair to face you as you approach.`

> 			`	"Well, as you may have surmised," he nods toward his covered feet, that were until recently, chained, "I am someone else's property."`

Reviewer: ```suggestion
			`	"Well, as you may have surmised," he nods toward his covered feet, that were, until recently, chained, "I am someone else's property."`
```
Alternatively (because this is a lot of commas):
```suggestion
			`	"Well, as you may have surmised," he nods toward his covered feet that were until recently chained, "I am someone else's property."`
```

> 			`	The injured man sighs, and looks away from you, "They let me do my own work, follow my own interests, pretend to be my own man, so long as I stay put, and do whatever they ask, when they ask it."`

Reviewer: ```suggestion
			`	The injured man sighs, and looks away from you, "They let me do my own work, follow my own interests, pretend to be my own man, so long as I stay put, and do whatever they ask, when they ask it.`
```

			action
				payment 10000000
			`	Weland wheels away from you, leaving you alone on the muddy street.`
				decline
			label no
			`	Weland frowns. "Thought you a little less common than that. Well, my mistake."`
			`	He pulls out a keypad, and hits a few buttons. "I can't get your stone back, but least I can do is pay you the 3 million I promised. I'm right sorry there isn't more I can do, but, they still own me, and there is only so much I can do."`
			action
				payment 3000000
			`	Weland nods at you, and then wheels away, back toward his shop, leaving you alone on the muddy street.`
				decline
			label doit
			`	Weland grins. "I never took you for a lily livered captain of a paper ship, glad to see I was right."`
			`	"The ship's called the 'Aotrou', and it's captained by Corentin Gwesklin, of the Gwesklin Slaver-Dynasty. That family tricked me, and has held me in bondage longer than most folk's memories; though there is little enough you can do 'bout that now."`

> 			`	Weland opens up a palm-display with star coordinates on it. "They are part of the Core Mining Concern. Semi-Sanctioned criminals, slavers, and all around bad guys. Led by the likes of the Gwesklin, and other hereditary slaver families. Though they can travel through civilized space unmolested, they do like to lay low in the Naper system, especially after altercations. That's where I'd wager you'd find Mr. Gwesklin, his ship, and the stone he stole from us.`

Reviewer: ```suggestion
			`	Weland opens up a palm-display with star coordinates on it. "They are part of the Core Mining Concern. Semi-sanctioned criminals, slavers, and all around bad guys. Led by the likes of the Gwesklin, and other hereditary slaver families. Though they can travel through civilized space unmolested, they do like to lay low in the Naper system, especially after altercations. That's where I'd wager you'd find Mr. Gwesklin, his ship, and the stone he stole from us.`
```

			`	"Be warned, he most likely will scan as a friendly, the C.M.C. leadership don't find it profitable to run pirate transponders. So keep your eyes open. Now, I think that's 'bout all needs being said at this juncture, just be sure to come back here once you got the stone. I got a surprise for ya."`
			`	Weland leans forward, and pats you on the arm, before turning his wheelchair around, and heading back in the direction of his shop. After a moment you're alone on a muddy street corner.`
				accept

	npc board
		government "Bounty (Disguised)"
		system "Naper"
		personality entering staying target timid uninterested
		ship "Falcon (Corentin Variant)" "C.M.C.S. Aotrou"

> 		dialog `It looks like Captain Gwesklin took an escape pod, and abandoned the ship earlier in the battle. The stone is still here, though.`

Reviewer: ```suggestion
		dialog `It looks like Captain Gwesklin took an escape pod, and abandoned the ship earlier in the battle. The stone is still here, though. You should take it.`
```

mission "Stone Harmonization 4surtr"
	source "Wayfarer"
	invisible
	to offer
		has "Stone Harmonization 3surtr: done"
	on offer
		require "Blood of Surtr"
		payment 3000000
		outfit "Blood of Surtr" -1
		outfit "Blood of Surtr (Harmonized)" 1
		conversation
			`Weland's laboratory has mostly been cleaned up, and repaired. Weland is waiting for you at his work bench, and gently places the Blood of Surtr on some sort of device.`
			`	"It sings." Weland almost whispers, as a display shows acoustic waves emanating from within the stone. "The crystal lattice within has been re-arranged in ways no one has ever dreamt of, in order to generate and produce waves that appear to alter the state of matter."`
			`	Weland shakes his head, "Not only is far advanced beyond anything humans could create, or recreate, but its an entire untapped field of study." Weland almost trembles with awe, "It's remarkable. Neither we, nor humans, nor any other race I know of, has deigned to interact with rock, crystal, and matter in this fashion."`
			choice

> 				`	"What do you mean, "we", aren't you human?"`

Reviewer: ```suggestion
				`	"What do you mean, 'we'? Aren't you human?"`
```

			`As you wander through the spaceport you notice someone is following you.`
			choice
				`	(Confront them.)`
					goto confront
				`	(Try to lose them.)`
			`	You bob and weave through crowds, go up and down stairs, and finally duck into a vintage electronic shop. You notice the owner are selling an ancient lime green computer called an iMac. The clerk see you looking at it and explains to you it just has one working application, a game called Escape Velocity Nova. By now it's clear you've lost your tail. So you thank the clerk and return to the spaceport.`
				defer
			label confront
			`	You turn to confront your pursuers. They quickly catch up to you. It's two men led by a tall hard-faced gray hard woman.`
			`	"Captain <last>, I'm Detective Song of the Oxford-Harvard Campus police and I'd like a moment of your time to discuss a recent incident on campus. Would you please follow me to a private room?"`
			choice
				`	"Sure, I've got nothing to hide."`
					goto hide
				`	"I demand a lawyer."`
				`	"Sorry, I am really quite busy right now. Could we do this later?"`
			`	She gives you a patronizing smile. "Absolutely. However, I really need you to answer a few questions."`

> 			`	The two men with her each grab one of your arms. "Don't worry, it'll quick and informal."`

Reviewer: ```suggestion
			`	The two men with her each grab one of your arms. "Don't worry, it'll be quick and informal."`
```

			`	They lead you to a small unadorned room in the spaceport. Detective Song and you enter it while the two men stand guard outside. There are two chairs in the room. Detective Song takes one and politely offers you the other.`
				goto start
			label hide
			`	"Excellent. I promise to make this brief. Now please follow me." She leads you to a small unadorned room in the spaceport. Detective Song and you enter it while the two men stand guard outside. There are two chairs in the room. Detective Song takes one and politely offers you the other.`
			label start
			choice
				`	(Take a seat.)`
				`	(Stand.)`
			`	She fishes through her pocket, pulls out a picture and hands it to you. "Do you recognize this man?"`

> 			`	It's Professor-Doctor Hullen-Freeze. He's standing in front of a some university wing named after him.`

Reviewer: ```suggestion
			`	It's Professor-Doctor Hullen-Freeze. He's standing in front of a university wing named after him.`
```

			choice
				`	"Yes. I met with him sometime back."`
					goto truth
				`	"No. I've never seen him in my life."`
				`	"I refuse to answer without my lawyer present."`
			`	"Huh. That's so interesting, because we have footage of you entering the building he works in." She pulls out a small screen that shows you, Tavish and Kormak walking into a building.`
			`	"Are you sure you didn't see him inside? Perhaps you just passed him in the hall?" She looks at you with an open concerned expression.`
			choice

> 				`	"Oh, that Dr. Hullen-Freeze. Yes I did meet with him."`

Reviewer: ```suggestion
				`	"Oh, that Dr. Hullen-Freeze. Yes, I did meet with him."`
```

					goto truth
				`	"I enter lots of buildings. Doesn't mean I talk to everyone in them."`
				`	"I refuse to answer without my lawyer present."`

> 			`	"Maybe you ran into his executive secrety? A Ms. Starla Stearn?" She looks at you hopefully.`

Reviewer: ```suggestion
			`	"Maybe you ran into his executive secretary? A Ms. Starla Stearn?" She looks at you hopefully.`
```

			choice
				`	"Okay. Yes, I did meet her, and I did see Dr. Hullen-Freeze."`
					goto truth
				`	"Hmm, no. Doesn't ring a bell. Sorry I can't be more help."`
				`	"I refuse to answer without my laywer present."`
			`	"You see the reason I ask is she has submitted a signed affidavit saying she met with you, two other men, and that you forced your way past her into Dr. Hullen-Freeze's office." She gives you a helpless expression, "Come on now, Captain <last>, I can't help you if you don't help yourself."`
			choice
				`	"Okay, okay. Yes I met her and him, but I didn't force my way anywhere. She led me to his office."`
					goto truth
				`	"Sorry. She must have me confused for someone else. I just was in that building to use the restroom."`
				`	"I refuse to answer without my lawer present."`
			`	Detective Song rubs the bridge of her nose and sighs. "Fine. You can go. I am afraid you have not been terribly helpful. If you can think of anything that may help our investigation please let us know." She knocks on the door and one of the men opens it.`

> 			`	"We're done here." She tells him, and they let you back out in the spaceport.`

Reviewer: ```suggestion
			`	"We're done here," she tells him, and they let you back out in the spaceport.`
```

				decline
			
			label truth
			`	"Ah, excellent. I really appreciate the cooperation. Could you tell me exactly what happened when you met with Ms. Starla and Dr. Hullen-Freeze?"`
			choice
				`	"Starla led us to his office where we briefly spoke and then left. Dr. Tavish was an old student of his and wanted to visit him."`
					goto liebaby
				`	"I thought I was escorting a former student of his to his office, but then Tavish and Kormak went crazy and Kormak killed him. They said they'd come after me next if I said anything."`
			`	Her notebook is out in a flash. "That's Doctor Julian Tavish, and I don't really have any records on this Kormak individual, what was his full name?"`
			choice

> 				`	"Gluniairn mac Kormakr, he's from the Deep. But I am not sure where exactly."`

Reviewer: ```suggestion
				`	"Gluniairn mac Kormak, he's from the Deep. But I am not sure where exactly."`
```

Author: So, it is supposed to be Kormakr, but everyone calls him Kormak.

Reviewer: Alright. I had been the last changer on that line, and thought I may have added it accidentally.

			choice
				`	"Tavish ordered Kormak to do it."`
				`	"Tavish did it."`
				`	"Kormak did it."`
			action
				event "prisontavish" 200
			`	"Perfect." she says, "Perfect." She closes her notebook.`
			`	"Julian Tavish is a known element here. He's thief and a conman who we've locked up more than a few times. Some years back Hullen-Freeze gave him a shot. Let him work for him. In the end Julian couldn't hack it going straight and ran off to run some new scams. I am sorry that you got caught up in his vendetta, but with this information we'll be able to finally take him down." She abruptly stands up.`
			`	"Thank you for your time, Captain <last>." She knocks on the door and one of the men opens it.`
			`	"You're free to go," she says, and they let you back out into the spaceport.`
				decline
			
			label liebaby
			`	"That's it? You didn't see or notice anything unusual? There were no arguments or tension between Hullen-Freeze and Julian Tavish?"`
			choice
				`	"That's it. It was cordial and we left. Starla was acting odd though. You might want to check up on her."`
				`	"Nothing else happened. We went in and we left. I saw nothing suspicious."`

> 			`	Detective Song sighs and stands up, "Very well. You've been most helpful. If you remember anything, anything at all, please do not hesitate to contact my office here on Chiron. Especially if it relates to Dr. Julian Tavish."`

Reviewer: ```suggestion
			`	Detective Song sighs and stands up, "Very well. You've been most helpful. If you remember anything, anything at all, please do not hesitate to contact my office here on Chiron. Especially if it relates to Dr. Julian Tavish.`
```

	on offer
		event "stone waiting most" 14
		fail

mission "Weland: Dragon Tech"
	name "Return to Weland on Wayfarer"
	description "Weland is done working on your artifact, and has requested that you return to Wayfarer and meet him."
	source
		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
	destination "Wayfarer"
	to offer
		has "event: welandwait2"
	on offer
		conversation
			`Your inbox blinks with a new message from Weland on Wayfarer.`
			`	<name>, this is Weland. Just letting you know your artifact is ready.`
			`	Come pick it up and pay for it at your earliest convenience.`
			`				-Xeno-Smith`
			`					Weland`

> 			`	P.S. Make sure you have 5 tons of outfit space!`

Reviewer: Why does he ask about 5 tons?
The Retrofitted Fusion Afterburner is 70 and 59 tons of outfit and weapon capacity, respectively.

Author: This may have undergone some changes since the original version of what was planned. So should it say to make sure you have 59 tons of outfit space?

Reviewer: Well, it takes up 70 tons of outfit space, but 59 tons of engine capacity.
So probably change it to say 59 and update the afterburner as well.

				accept
	on complete
		conversation
			`You find Weland inside his makeshift laboratory. Your artifact, covered in cords and various machinery, is no longer dull and inert, but has whirred to life.`
			`	"Take a moment. It's a rare sight, alright. Watchin' somethin's been dormant for longer than mankind's had words spur to life before ya, why, it's enough to give a man religion, or something near abouts." Weland pats you on the back.`

> 			`	"Now, what we have here is an Fusion Afterburner, it's an afterburner. From what I can tell, despite it's age, a pretty effective one too."`

Author: looks like an edit left both versions in... either works here.

			`	"But, here you are flying free with one on your ship doin' whatever it is these strange stones do. I'd as pay a pretty penny for somethin' like that. How about I waive the fee for fixin' up your artifact, and pay you 10 million credits for that 'Tear of Skade' stone of yours?"`

			label tearchoice
			choice
				`	(Refuse to sell the Tear of Skade.)`
					goto refuseskade
				`	(Agree to sell the Tear of Skade.)`
					goto sellskade

			label tearsell
			`	"Now that we got that all settled. There's one other item I was wantin' to discuss. Somethin' in the ship schematics you sent over caught my eye. I think you had it labeled, 'Tear of Skade'. Now, if that is what I think it is, well, it's worth a lot of money. I never met nobody that has laid eyes on a Deep Stone, let alone owned one. It's phantom tech kind of stuff you hear about from spooks, and conspiracy nuts. Most say it isn't even real, and if it is, only a few ever been found, and those the government holds under lock and key.`
			`	"But, here you are flying free with one on your ship doin' whatever it is these strange stones do. I'd offer to buy it off ya, but you just took most my cash.`

> 			`	"Well, what if I borrowed the stone? Just, hear me out, alright. Give me three months to study it, learn what secrets from it I can. I could probably scrounge up another 3 million to pay you for the pleasure. After 3 months, you come collect the stone, and your payment. Everyone wins. Whatd'ya say?"`

Reviewer: ```suggestion
			`	"Well, what if I borrowed the stone? Just, hear me out, alright. Give me three months to study it, learn what secrets from it I can. I could probably scrounge up another three million to pay you for the pleasure. After three months, you come collect the stone, and your payment. Everyone wins. Whatd'ya say?"`
```

				goto tearchoice
			
			label stonekeep

> 			`	"Now that we got that all settled. There's one other item I was wantin' to discuss. Somethin' in the ship schematics you sent over caught my eye, two of 'em in fact. I think you had them labeled, 'Tear of Skade', and 'Blood of Surtr. Now, if they are what I think they are, well, they're worth a lot of money. I never met nobody that has laid eyes on a Deep Stone, let alone owned one. It's phantom tech, kind of stuff you hear about from spooks, and conspiracy nuts. Most say it isn't even real, and if it is, only a few ever been found, and those the government holds under lock and key.`

Reviewer: ```suggestion
			`	"Now that we got that all settled. There's one other item I was wantin' to discuss. Somethin' in the ship schematics you sent over caught my eye, two of 'em in fact. I think you had them labeled 'Tear of Skade' and 'Blood of Surtr.' Now, if they are what I think they are, well, they're worth a lot of money. I never met nobody that has laid eyes on a Deep Stone, let alone owned one. It's phantom tech, kind of stuff you hear about from spooks, and conspiracy nuts. Most say it isn't even real, and if it is, only a few have ever been found, and those the government holds under lock and key.`
```

> 			`	"But, here you are, flying free with not one, but two, on your ship, doin' whatever it is these strange stones do. I'd as pay a pretty penny for somethin' like that. How about I waive the fee for fixin' up your artifact, and pay you 10 million credits for one of those stones of yours?"`

Reviewer: ```suggestion
			`	"But, here you are, flying free with not one, but two, on your ship, doin' whatever it is these strange stones do. I'd pay a pretty penny for somethin' like that. How about I waive the fee for fixin' up your artifact, and pay you ten million credits for one of those stones of yours?"`
```

			label stonechoice
			choice
				`	(Refuse to sell either stone.)`
					goto refuse
				`	(Agree to sell the Tear of Skade.)`
					goto sellskade
				`	(Agree to sell the Blood of Surtr.)`
					goto sellsurtr
			
			label stonesell
			`	"Now that we got that all settled. There's one other item I was wantin' to discuss. Somethin' in the ship schematics you sent over caught my eye, two of 'em in fact. I think you had them labeled, 'Tear of Skade', and 'Blood of Surtr. Now, if they are what I think they are, well, they're worth a lot of money. I never met nobody that has laid eyes on a Deep Stone, let alone owned one. It's phantom tech, kind of stuff you hear about from spooks, and conspiracy nuts. Most say it isn't even real, and if it is, only a few ever been found, and those the government holds under lock and key.`
			`	"But, here you are flying free with not one, but two, on your ship doin' whatever it is these strange stones do. I'd offer to buy one of 'em off ya, but you just took most my cash.`

> 			`	"Well, what if I borrowed one of the stones? Just, hear me out, alright. Give me three months to study it, learn what secrets from it I can. I could probably scrounge up another 3 million to pay you for the pleasure. After 3 months, you come collect the stone, and your payment. Everyone wins. Whatd'ya say?"`
> 				goto stonechoice
> 			
> 			label refuse
> 			action
> 				payment -6000000
> 			`	"Well, what if I borrowed one of the stones? Just, hear me out, alright. Give me three months to study it, learn what secrets from it if I can. I'd pay 3 million credits for the pleasure. After 3 months, you come collect the stone, and your payment. Everyone wins. Whatd'ya say?"`

Reviewer: ```suggestion
			`	"Well, what if I borrowed one of the stones? Just, hear me out, alright. Give me three months to study it, learn what secrets from it I can. I could probably scrounge up another three million to pay you for the pleasure. After three months, you come collect the stone, and your payment. Everyone wins. Whatd'ya say?"`
				goto stonechoice
			
			label refuse
			action
				payment -6000000
			`	"Well, what if I borrowed one of the stones? Just, hear me out, alright. Give me three months to study it, learn what secrets from it if I can. I'd pay three million credits for the pleasure. After three months, you come collect the stone, and your payment. Everyone wins. Whatd'ya say?"`
```

			`	Crestfallen, Weland turns away from you. "They're your stones, your call."`
			`	He says nothing more and limps off into some back room, chains dragging after him. You show yourself out.`
				decline
			
			label studyskade
			action
				outfit "Tear of Skade" -1
				event "welandstone" 90
				set "weland: study skade"
			`	Weland grins. "A little time's better than no time. Put 'er there."`
			`	He takes your hand and shakes it with great enthusiasm. "Imagine what secrets await."`
			`	Later on you transfer over the Tear of Skade to a feverishly eager Weland. You promise to back in three months, for stone and payment.`
				decline
			
			label sellskade
			action
				payment 10000000
				outfit "Tear of Skade" -1
			`	Weland grins like a hyena, "Hot damn. Know, that this is just about every red cent I own, and in some cases don't. But with this stone, the secrets it holds, who knows what I'll discover."`
			`	He laughs, maniacally, and transfers the credits over to you.`
			`	He hobbles over to you, and shakes your hand. "Pleasure doin' business. And perhaps we'll run into each other again."`
				decline
			
			label refuseskade
			action
				payment -6000000

> 			`	"Well, what if I borrowed the stone? Just, hear me out, alright. Give me three months to study it, learn what secrets from it if I can. I'd pay 3 million credits for the pleasure. After 3 months, you come collect the stone, and your payment. Everyone wins. Whatd'ya say?"`

Reviewer: ```suggestion
			`	"Well, what if I borrowed the stone? Just hear me out, alright. Give me three months to study it, learn what secrets from it if I can. I'd pay three million credits for the pleasure. After three months, you come collect the stone, and your payment. Everyone wins. Whatd'ya say?"`
```

				`	(Refuse him.)`
			`	Crestfallen, Weland turns away from you. "It's your stone, your call."`
			`	He says nothing more and limps away into some back room, chains dragging after him. You show yourself out.`
				decline

			label studysurtr
			action
				event "welandstone" 90
				outfit "Blood of Surtr" -1
				set "weland: study surtr"
			`	Weland grins. "A little time's better than no time. Put 'er there."`
			`	He takes your hand and shakes it with great enthusiasm. "Imagine what secrets await."`
			`	Later on you transfer over the Blood of Surtr to a feverishly eager Weland. You promise to back in three months, for stone and payment.`
				decline
			
			label sellsurtr
			action
				payment 10000000
				outfit "Blood of Surtr" -1
			`	Weland grins like a hyena, "Hot damn. Know that this is just about every red cent I own, and in some cases don't. But with this stone, the secrets it holds, who knows what I'll discover."`
			`	He laughs, maniacally, and transfers the credits over to you.`
			`	He hobbles over to you, and shakes your hand. "Pleasure doin' business. And perhaps we'll run into each other again."`
				decline

			label refusesurtr
			action
				payment -6000000

> 			`	"Well, what if I borrowed the stone? Just, hear me out, alright. Give me three months to study it, learn what secrets from it if I can. I'd pay 3 million credits for the pleasure. After 3 months, you come collect the stone, and your payment. Everyone wins. Whatd'ya say?"`

Reviewer: ```suggestion
			`	"Well, what if I borrowed the stone? Just hear me out, alright. Give me three months to study it, learn what secrets from it if I can. I'd pay three million credits for the pleasure. After three months, you come collect the stone, and your payment. Everyone wins. Whatd'ya say?"`
```

			choice
				`	(Let Weland study the stone.)`
					goto studysurtr
				`	(Refuse him.)`
			`	Crestfallen, Weland turns away from you. "It's your stone, your call."`
			`	He says nothing more and limps away into some back room, chains dragging after him. You show yourself out.`
				decline

event "welandstone"

mission "Weland: Stone Harmonization"

> 	source

Reviewer: ```suggestion
	name "Visit Weland"
	description "Visit Weland on <destination> to speak with him about something."
	source
```

		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
	destination "Wayfarer"
	landing
	to offer
		has "Weland: Dragon Tech: offered"
		has "event: welandstone"
		random < 10
	on offer
		require "Tear of Skade"
		require "Blood of Surtr"
		dialog
			`Your inbox blinks with a new message from Weland on Wayfarer.`
			`	<name>, I was wanting to speak with you about something.`
			`	Please stop by my shop on Wayfarer at your earliest convenience.`

> 			`				-Xeno-Smith`

Reviewer: ```suggestion
			`				- Xeno-Smith`
```

			`					Weland`
	on complete
		conversation
			`You find Weland tinkering away in his shop. He puts his inscrutable tools down and waves to you as you enter.`
			`	"Ah, there you are. I am sure you're wonderin' yourself sick trying to divine why I called ya." He grins, and limps toward you, dragging metal chains behind his feet.`
			`	"I heard word, from someone who's done crew work on your ship, that you're in possession of a couple of Deep Stones." He nearly whispers the last the two words.`

> 			`	"I've been told you call 'em, the 'Tear of Skade', and the 'Blood of Surtr'". He watches your eyes carefully as he names off the stones. "Now, if they are what I think they are, well, they're worth a lot of money. I never met nobody that has laid eyes on a Deep Stone, let alone owned one. It's phantom tech, kind of stuff you hear about from spooks, and conspiracy nuts. Most say it isn't even real, and if it is, only a few ever been found, and those the government holds under lock and key.`

Reviewer: ```suggestion
			`	"I've been told you call 'em the 'Tear of Skade' and the 'Blood of Surtr.'". He watches your eyes carefully as he names off the stones. "Now, if they are what I think they are, well, they're worth a lot of money. I never met nobody that has laid eyes on a Deep Stone, let alone owned one. It's phantom tech, kind of stuff you hear about from spooks, and conspiracy nuts. Most say it isn't even real, and if it is, only a few have ever been found, and those the government holds under lock and key.`
```

			`	"But, here you are, flying free with not one, but two, on your ship, doin' whatever it is these strange stones do. I'd offer to buy one of 'em off ya, but I don't have anywhere nears enough cash for one of those.`

> 			`	"So, what if I borrowed one of the stones? Just, hear me out, alright. Give me three months to study it, learn what secrets from it I can. I'd pay you 3 million credits for the pleasure. After 3 months, you come collect the stone, and your payment. Everyone wins. Whatd'ya say?"`

Reviewer: ```suggestion
			`	"So, what if I borrowed one of the stones? Just hear me out, alright. Give me three months to study it, learn what secrets from it I can. I'd pay you three million credits for the pleasure. After three months, you come collect the stone, and your payment. Everyone wins. Whatd'ya say?"`
```

				accept

	npc board
		government "Bounty (Disguised)"
		system "Naper"
		personality entering staying target timid uninterested
		ship "Falcon (Corentin Variant)" "C.M.C.S. Aotrou"
		dialog `It looks like Captain Gwesklin took an escape pod and abandoned the ship earlier in the battle. The stone is still here, though. You should take it.`
	on visit
		dialog `You haven't dealt with the Aotrou.`
	on complete
		conversation
			`Weland is waiting for you when you return. He's still wheelchair bound, but otherwise looking surprisingly better. He also doesn't seem too upset that Captain Corentin Gwesklin escaped. He tells you to quickly meet him at his shop with your reclaimed stone.`

mission "Weland: Stone Retrieved Harmonization"
	source "Wayfarer"
	invisible
	to offer
		or
			has "Weland: Retrieve Skade: done"
			has "Weland: Retrieve Surtr: done"
		or
			has "outfit: Blood of Surtr"
			has "outfit: Tear of Skade"
	on offer
		payment 3000000
		conversation
			branch skade
				not "outfit: Blood of Surtr"
				has "outfit: Tear of Skade"

> 			`Weland's laboratory has mostly been cleaned up, and repaired. Weland is waiting for you at his work bench, and gently places the Blood of Surtr on some sort of device.`

Reviewer: ```suggestion
			`Weland's laboratory has mostly been cleaned up, and repaired. Weland is waiting for you at his work bench and gently places the Blood of Surtr on some sort of device.`
```

				goto singingstones
			label skade

> 			`Weland's laboratory has mostly been cleaned up, and repaired. Weland is waiting for you at his work bench, and gently places the Tear of Skade on some sort of device.`

Reviewer: ```suggestion
			`Weland's laboratory has mostly been cleaned up, and repaired. Weland is waiting for you at his work bench and gently places the Tear of Skade on some sort of device.`
```

			choice
				`	"What do you mean, 'we'? Aren't you human?"`
			`	The man's head snaps up to look at you through briefly narrowed eyes before he smiles warmly, "Ah, well, just a slip of tongue. I was so excited there, I was babblin' nonsense. Of course I'm human." He laughs.`
			`	"But, this is more than just academic for you, because I do believe this stone has fallen out of, what I can only call, harmony. I didn't have a chance before, but now, by projecting just the right sound waves back at the stone..."`
			`	Weland fiddles with his controls for a bit. "... I can return the stone to harmony."`
			branch skade2
				not "outfit: Blood of Surtr"
				has "outfit: Tear of Skade"
			`	He hits a button, and it feels like for just a second the air is sucked out of the room into the stone, followed by a forceful gust of crackling energized air.`
			`	"Ah ha! Hot damn, it worked!" He studies the stone for a second, taking some readings, than hands it back to you.`
			`	"See for yourself, it's more energetic than it used to be, more powerful. Harmonized." Weland laughs to himself, and then muses. "Crystalline tonal architecture, ha, who woulda thought, alright."`

> 			`	Weland opens up a keypad, and begins tapping it. "I do believe I still owe you 3 million credits, which I have just transferred to your account. Also, consider your newly harmonized stone as a bonus for unfortunate circumstances.`

Reviewer: ```suggestion
			`	Weland opens up a keypad, and begins tapping it. "I do believe I still owe you three million credits, which I have just transferred to your account. Also, consider your newly harmonized stone as a bonus for unfortunate circumstances.`
```

> 			branch skade
> 				not "outfit: Blood of Surtr"
> 				has "outfit: Tear of Skade"
> 			`	He hits a button, and it feels like for just a second the air is sucked out of the room into the stone, followed by a forceful gust of crackling energized air.`
> 			`	"Ah ha! Hot damn, it worked!" He studies the stone for a second, taking some readings, than hands it back to you.`
> 			`	"See for yourself, it's more energetic than it used to be, more powerful. Harmonized." Weland laughs to himself, and then muses. "Crystalline tonal architecture, ha, who woulda thought, alright."`
> 			`	Weland opens up a keypad, and begins tapping it. "I do believe I still owe you 3 million credits, which I have just transferred to your account. Also, consider your newly harmonized stone as a bonus for unfortunate circumstances.`
> 				goto questions
> 			label skade

Reviewer: ```suggestion
			branch skade2
				not "outfit: Blood of Surtr"
				has "outfit: Tear of Skade"
			`	He hits a button, and it feels like for just a second the air is sucked out of the room into the stone, followed by a forceful gust of crackling energized air.`
			`	"Ah ha! Hot damn, it worked!" He studies the stone for a second, taking some readings, than hands it back to you.`
			`	"See for yourself, it's more energetic than it used to be, more powerful. Harmonized." Weland laughs to himself, and then muses. "Crystalline tonal architecture, ha, who woulda thought, alright."`
			`	Weland opens up a keypad, and begins tapping it. "I do believe I still owe you 3 million credits, which I have just transferred to your account. Also, consider your newly harmonized stone as a bonus for unfortunate circumstances.`
				goto questions
			label skade2
```
oops

			`	He hits a button, and it feels like for just a second the air is sucked out of the room into the stone, followed by a forceful gust of cold air.`
			`	"Ah ha! Cold damn, it worked!" He studies the stone for a second, taking some readings, than hands it back to you.`
			`	"See for yourself, it's colder than it used to be, more powerful. Harmonized." Weland laughs to himself, and then muses. "Crystal tonal architecture, ha, who woulda thought, alright."`

> 			`	Weland opens up a keypad, and begins tapping it. "I do believe I still owe you 3 million credits, which I have just transferred to your account. Also, consider your newly harmonized stone as a bonus for unfortunate circumstances.`

Reviewer: ```suggestion
			`	Weland opens up a keypad, and begins tapping it. "I do believe I still owe you three million credits, which I have just transferred to your account. Also, consider your newly harmonized stone as a bonus for unfortunate circumstances.`
```

				decline
			label next
			`	The injured man gets a distant look in his eyes, "Well, I figure I'll just get back to doin' what I am doin'. Lots of things to figure, lots of cords to sell. Always work, and excitement for someone in my boots. I wager one of these days I'll shake loose of my masters. But until then I'll abide."`
				goto questions
			label harmonize
			`	Weland looks at you wistfully, and then, with a tinge of sadness, shakes his head. "I wager it's a sight too dangerous for that. Each stone is grown, or sung differently, it would take me months to figure out how to harmonize another one. And in the meantime, I am still watched, still unable to leave, still the property of others. If you left somethin' that valuable with me again, they'd take it, and this time to somewhere you wouldn't find it."`
			`	He shrugs, "Perhaps, some other time. On a different world, in different circumstances. But, not now, not here, and not like this."`
				goto questions
			label offworld

> 			`	He gives you a patronizing smile. "Folks that did this to me. They've been doin' this kind of work for a while, and I, well, I am like the crown jewel of their collection. They have hooks into me, inside of me, all around me, that prevent me from leaving, at least alive. They like to give me the illusion of space, pretend they are... gentile 'bout the whole thing. But make no mistake, I am not free, and I cannot go."`

Reviewer: "gentile"? Was there another word supposed to be here instead?

Author: genteel, sorry.

	on offer
		event "stone waiting most" 14
		fail

mission "Weland: Dragon Tech"
	name "Return to Weland on Wayfarer"
	description "Weland is done working on your artifact, and has requested that you return to Wayfarer and meet him."
	source
		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
	destination "Wayfarer"
	to offer
		has "event: welandwait2"
	on offer
		conversation
			`Your inbox blinks with a new message from Weland on Wayfarer.`
			`	<name>, this is Weland. Just letting you know your artifact is ready.`
			`	Come pick it up and pay for it at your earliest convenience.`
			`				- Xeno-Smith`
			`					Weland`

> 			`	P.S. Make sure you have 59 tons of outfit space!`

Reviewer: ```suggestion
			`	P.S.: Make sure you have 59 tons of outfit space!`
```

			branch nostone
				not "outfit: Tear of Skade"
				not "outfit: Blood of Surtr"
			
			choice
				`	(Keep the Fusion Afterburner.)`
				`	(Sell the Fusion Afterburner.)`
					goto sell
			action
				outfit "Retrofitted Fusion Afterburner" 1
			`	"Ah, well, was worth a shot. Still, had 6 months to study it, so it's not a total bust. I'll have it dropped off at your ship, assumin' you have the space." He hobbles over to you and shakes your hand.`
			`	"I'll miss having it around. Made the place feel mysterious." He shrugs.`

			branch bloodkeep
				has "outfit: Blood of Surtr"
				not "outfit: Tear of Skade"
			branch tearkeep
				has "outfit: Tear of Skade"
				not "outfit: Blood of Surtr"
			branch stonekeep
				has "outfit: Tear of Skade"
				has "outfit: Blood of Surtr"

			label sell
			action
				payment 4000000
			`	Weland looks feverish. "Hot damn. Know that this is just about every red cent I own, and in some cases don't. But I think it'll be worth it."`

> 			`	He hobbles over to you, and shakes your hand. "Pleasure doin' business."`

Reviewer: ```suggestion
			`	He hobbles over to you, and shakes your hand. "Pleasure doin' business." He shrugs.`
```

			branch bloodsell
				has "outfit: Blood of Surtr"
				not "outfit: Tear of Skade"
			branch tearsell
				has "outfit: Tear of Skade"
				not "outfit: Blood of Surtr"
			branch stonesell
				has "outfit: Tear of Skade"
				has "outfit: Blood of Surtr"
			
			label nostone
			choice
				`	(Keep the Fusion Afterburner.)`
					goto nostonekeep
				`	(Sell the Fusion Afterburner.)`
			action
				payment 40000000
			`	Weland looks feverish. "Hot damn. Know that this is just about every red cent I own, and in some cases don't. But I think it'll be worth it."`
			`	He hobbles over to you, and shakes your hand. "Pleasure doin' business. And perhaps we'll run into each other again."`
				decline

> 			label nostonekeep

Reviewer: ```suggestion

			label nostonekeep
```

			action
				outfit "Retrofitted Fusion Afterburner" 1
				payment -6000000
				event "welandstone" 90
			`	"Ah, well, was worth a shot. Still, had 6 months to study it, so it's not a total bust. I'll have it dropped off at your ship, assumin' you have the space." He hobbles over to you, and shakes your hand.`
			`	"I'll miss having it around. Made the place feel mysterious." You say your goodbyes, and head back to your ship.`
				decline
			
			label bloodkeep

> 			`	"Now that we got that all settled. There's one other item I was wantin' to discuss. Somethin' in the ship schematics you sent over caught my eye. I think you had it labeled 'Blood of Surtr.' Now, if that is what I think it is, well, it's worth a lot of money. I never met nobody that has laid eyes on a legendary Deep Stone, let alone owned one. It's phantom tech, kind of stuff you hear about from spooks, and conspiracy nuts. Most say it isn't even real, and if it is, only a few have ever been found, and those the government holds under lock and key.`

Reviewer: ```suggestion
			`	"Now that we got that all settled. There's one other item I was wantin' to discuss. Somethin' in the ship schematics you sent over caught my eye. I think you had it labeled 'Blood of Surtr.' Now, if that is what I think it is, well, it's worth a lot of money. I never met nobody that has laid eyes on a legendary Deep Stone, let alone owned one. It's phantom tech, kind of stuff you hear about from spooks and conspiracy nuts. Most say it isn't even real, and if it is, only a few have ever been found, and those the government holds under lock and key.`
```

			`	"But, here you are, flying free with one on your ship, doin' whatever it is these strange stones do. I'd pay a pretty penny for somethin' like that. How about I waive the fee for fixin' up your artifact, and pay you ten million credits for that 'Blood of Surtr' stone of yours?"`

			label bloodchoice
			choice
				`	(Refuse to sell the Blood of Surtr.)`
					goto refusesurtr
				`	(Agree to sell the Blood of Surtr.)`
					goto sellsurtr
			
			label bloodsell

> 			`	"Now that we got that all settled. There's one other item I was wantin' to discuss. Somethin' in the ship schematics you sent over caught my eye. I think you had it labeled 'Blood of Surtr.' Now, if that is what I think it is, well, it's worth a lot of money. I never met nobody that has laid eyes on a legendary Deep Stone, let alone owned one. It's phantom tech kind of stuff you hear about from spooks, and conspiracy nuts. Most say it isn't even real, and if it is, only a few have ever been found, and those the government holds under lock and key.`

Reviewer: ```suggestion
			`	"Now that we got that all settled. There's one other item I was wantin' to discuss. Somethin' in the ship schematics you sent over caught my eye. I think you had it labeled 'Blood of Surtr.' Now, if that is what I think it is, well, it's worth a lot of money. I never met nobody that has laid eyes on a legendary Deep Stone, let alone owned one. It's phantom tech kind of stuff you hear about from spooks and conspiracy nuts. Most say it isn't even real, and if it is, only a few have ever been found, and those the government holds under lock and key.`
```

			`	"But, here you are flying free with one on your ship doin' whatever it is these strange stones do. I'd offer to buy it off ya, but you just took most my cash.`
			`	"Well, what if I borrowed the stone? Just, hear me out, alright. Give me three months to study it, learn what secrets from it I can. I could probably scrounge up another three million to pay you for the pleasure. After three months, you come collect the stone, and your payment. Everyone wins. Whatd'ya say?"`
				goto bloodchoice
						
			label tearkeep

> 			`	"Now that we got that all settled. There's one other item I was wantin' to discuss. Somethin' in the ship schematics you sent over caught my eye. I think you had it labeled 'Tear of Skade.' Now, if that is what I think it is, well, it's worth a lot of money. I never met nobody that has laid eyes on a Deep Stone, let alone owned one. It's phantom tech kind of stuff you hear about from spooks, and conspiracy nuts. Most say it isn't even real, and if it is, only a few have ever been found, and those the government holds under lock and key.`

Reviewer: ```suggestion
			`	"Now that we got that all settled. There's one other item I was wantin' to discuss. Somethin' in the ship schematics you sent over caught my eye. I think you had it labeled 'Tear of Skade.' Now, if that is what I think it is, well, it's worth a lot of money. I never met nobody that has laid eyes on a Deep Stone, let alone owned one. It's phantom tech kind of stuff you hear about from spooks and conspiracy nuts. Most say it isn't even real, and if it is, only a few have ever been found, and those the government holds under lock and key.`
```

			`	"But, here you are flying free with one on your ship doin' whatever it is these strange stones do. I'd pay a pretty penny for somethin' like that. How about I waive the fee for fixin' up your artifact, and pay you ten million credits for that 'Tear of Skade' stone of yours?"`

			label tearchoice
			choice
				`	(Refuse to sell the Tear of Skade.)`
					goto refuseskade
				`	(Agree to sell the Tear of Skade.)`
					goto sellskade

			label tearsell

> 			`	"Now that we got that all settled. There's one other item I was wantin' to discuss. Somethin' in the ship schematics you sent over caught my eye. I think you had it labeled 'Tear of Skade.' Now, if that is what I think it is, well, it's worth a lot of money. I never met nobody that has laid eyes on a Deep Stone, let alone owned one. It's phantom tech kind of stuff you hear about from spooks, and conspiracy nuts. Most say it isn't even real, and if it is, only a few have ever been found, and those the government holds under lock and key.`

Reviewer: ```suggestion
			`	"Now that we got that all settled. There's one other item I was wantin' to discuss. Somethin' in the ship schematics you sent over caught my eye. I think you had it labeled 'Tear of Skade.' Now, if that is what I think it is, well, it's worth a lot of money. I never met nobody that has laid eyes on a Deep Stone, let alone owned one. It's phantom tech kind of stuff you hear about from spooks and conspiracy nuts. Most say it isn't even real, and if it is, only a few have ever been found, and those the government holds under lock and key.`
```

			`	"But, here you are flying free with one on your ship doin' whatever it is these strange stones do. I'd offer to buy it off ya, but you just took most my cash.`
			`	"Well, what if I borrowed the stone? Just, hear me out, alright. Give me three months to study it, learn what secrets from it I can. I could probably scrounge up another three million to pay you for the pleasure. After three months, you come collect the stone, and your payment. Everyone wins. Whatd'ya say?"`
				goto tearchoice
			
			label stonekeep

> 			`	"Now that we got that all settled. There's one other item I was wantin' to discuss. Somethin' in the ship schematics you sent over caught my eye, two of 'em in fact. I think you had them labeled 'Tear of Skade' and 'Blood of Surtr.' Now, if they are what I think they are, well, they're worth a lot of money. I never met nobody that has laid eyes on a Deep Stone, let alone owned one. It's phantom tech, kind of stuff you hear about from spooks, and conspiracy nuts. Most say it isn't even real, and if it is, only a few have ever been found, and those the government holds under lock and key.`

Reviewer: ```suggestion
			`	"Now that we got that all settled. There's one other item I was wantin' to discuss. Somethin' in the ship schematics you sent over caught my eye, two of 'em in fact. I think you had them labeled 'Tear of Skade' and 'Blood of Surtr.' Now, if they are what I think they are, well, they're worth a lot of money. I never met nobody that has laid eyes on a Deep Stone, let alone owned one. It's phantom tech, kind of stuff you hear about from spooks and conspiracy nuts. Most say it isn't even real, and if it is, only a few have ever been found, and those the government holds under lock and key.`
```

			`	"But, here you are, flying free with not one, but two, on your ship, doin' whatever it is these strange stones do. I'd pay a pretty penny for somethin' like that. How about I waive the fee for fixin' up your artifact, and pay you ten million credits for one of those stones of yours?"`

			label stonechoice
			choice
				`	(Refuse to sell either stone.)`
					goto refuse
				`	(Agree to sell the Tear of Skade.)`
					goto sellskade
				`	(Agree to sell the Blood of Surtr.)`
					goto sellsurtr
			
			label stonesell

> 			`	"Now that we got that all settled. There's one other item I was wantin' to discuss. Somethin' in the ship schematics you sent over caught my eye, two of 'em in fact. I think you had them labeled, 'Tear of Skade', and 'Blood of Surtr. Now, if they are what I think they are, well, they're worth a lot of money. I never met nobody that has laid eyes on a Deep Stone, let alone owned one. It's phantom tech, kind of stuff you hear about from spooks, and conspiracy nuts. Most say it isn't even real, and if it is, only a few ever been found, and those the government holds under lock and key.`

Reviewer: ```suggestion
			`	"Now that we got that all settled. There's one other item I was wantin' to discuss. Somethin' in the ship schematics you sent over caught my eye, two of 'em in fact. I think you had them labeled, 'Tear of Skade', and 'Blood of Surtr. Now, if they are what I think they are, well, they're worth a lot of money. I never met nobody that has laid eyes on a Deep Stone, let alone owned one. It's phantom tech, kind of stuff you hear about from spooks and conspiracy nuts. Most say it isn't even real, and if it is, only a few ever been found, and those the government holds under lock and key.`
```

			`	"Well, what if I borrowed one of the stones? Just, hear me out, alright. Give me three months to study it, learn what secrets from it if I can. I'd pay three million credits for the pleasure. After three months, you come collect the stone, and your payment. Everyone wins. Whatd'ya say?"`
			choice
				`	(Let Weland study the Tear of Skade.)`
					goto studyskade
				`	(Let Weland study the Blood of Surtr.)`
					goto studysurtr
				`	(Refuse him.)`
			`	Crestfallen, Weland turns away from you. "They're your stones, your call."`
			`	He says nothing more and limps off into some back room, chains dragging after him. You show yourself out.`
				decline
			
			label studyskade
			action
				outfit "Tear of Skade" -1
				event "welandstone" 90
				set "weland: study skade"
			`	Weland grins. "A little time's better than no time. Put 'er there."`
			`	He takes your hand and shakes it with great enthusiasm. "Imagine what secrets await."`
			`	Later on you transfer over the Tear of Skade to a feverishly eager Weland. You promise to back in three months, for stone and payment.`
				decline
			
			label sellskade
			action
				payment 10000000
				outfit "Tear of Skade" -1

> 			`	Weland grins like a hyena, "Hot damn. Know, that this is just about every red cent I own, and in some cases don't. But with this stone, the secrets it holds, who knows what I'll discover."`

Reviewer: ```suggestion
			`	Weland grins like a hyena, "Hot damn. Know that this is just about every red cent I own, and in some cases don't. But with this stone, the secrets it holds, who knows what I'll discover."`
```

			`	<name>, I was wanting to speak with you about something.`
			`	Please stop by my shop on Wayfarer at your earliest convenience.`
			`				- Xeno-Smith`
			`					Weland`
	on complete
		conversation
			`You find Weland tinkering away in his shop. He puts his inscrutable tools down and waves to you as you enter.`
			`	"Ah, there you are. I am sure you're wonderin' yourself sick trying to divine why I called ya." He grins, and limps toward you, dragging metal chains behind his feet.`
			`	"I heard word, from someone who's done crew work on your ship, that you're in possession of a couple of Deep Stones." He nearly whispers the last the two words.`
			`	"I've been told you call 'em the 'Tear of Skade' and the 'Blood of Surtr.'". He watches your eyes carefully as he names off the stones. "Now, if they are what I think they are, well, they're worth a lot of money. I never met nobody that has laid eyes on a Deep Stone, let alone owned one. It's phantom tech, kind of stuff you hear about from spooks, and conspiracy nuts. Most say it isn't even real, and if it is, only a few have ever been found, and those the government holds under lock and key.`

> 			`	"But, here you are, flying free with not one, but two, on your ship, doin' whatever it is these strange stones do. I'd offer to buy one of 'em off ya, but I don't have anywhere nears enough cash for one of those.`

Reviewer: ```suggestion
			`	"But, here you are, flying free with not one, but two, on your ship, doin' whatever it is these strange stones do. I'd offer to buy one of 'em off ya, but I don't have anywhere near enough cash for one of those.`
```

			action
				payment 10000000
			`	Weland wheels away from you, leaving you alone on the muddy street.`
				decline
			label no
			`	Weland frowns. "Thought you a little less common than that. Well, my mistake."`
			`	He pulls out a keypad, and hits a few buttons. "I can't get your stone back, but least I can do is pay you the three million I promised. I'm right sorry there isn't more I can do, but, they still own me, and there is only so much I can do."`
			action
				payment 3000000
			`	Weland nods at you, and then wheels away, back toward his shop, leaving you alone on the muddy street.`
				decline
			label doit
			`	Weland grins. "I never took you for a lily livered captain of a paper ship, glad to see I was right.`
			`	"The ship's called the 'Aotrou', and it's captained by Corentin Gwesklin, of the Gwesklin Slaver-Dynasty. That family tricked me, and has held me in bondage longer than most folk's memories; though there is little enough you can do 'bout that now."`

> 			`	Weland opens up a palm-display with star coordinates on it. "They are part of the Core Mining Concern. Semi-sanctioned criminals, slavers, and all around bad guys. Led by the likes of the Gwesklin and other hereditary slaver families. Though they can travel through civilized space unmolested, they do like to lay low in the Naper system, especially after altercations. That's where I'd wager you'd find Mr. Gwesklin, his ship, and the stone he stole from us.`

Reviewer: ```suggestion
			`	Weland opens up a palm-display with star coordinates on it. "They are part of the Core Mining Concern. Semi-sanctioned criminals, slavers, and all around bad guys. Led by the likes of the Gwesklin and other hereditary slaver families. Though they can travel through civilized space unmolested, they do like to lay low in the <waypoints> system, especially after altercations. That's where I'd wager you'd find Mr. Gwesklin, his ship, and the stone he stole from us.`
```

			action
				payment 10000000
			`	Weland wheels away from you, leaving you alone on the muddy street.`
				decline
			label no
			`	Weland frowns. "Thought you a little less common than that. Well, my mistake."`
			`	He pulls out a keypad, and hits a few buttons. "I can't get your stone back, but least I can do is pay you the three million I promised. I'm right sorry there isn't more I can do, but, they still own me, and there is only so much I can do."`
			action
				payment 3000000
			`	Weland nods at you, and then wheels away, back toward his shop, leaving you alone on the muddy street.`
				decline
			label doit
			`	Weland grins. "I never took you for a lily livered captain of a paper ship, glad to see I was right."`
			`	"The ship's called the 'Aotrou', and it's captained by Corentin Gwesklin, of the Gwesklin Slaver-Dynasty. That family tricked me, and has held me in bondage longer than most folk's memories; though there is little enough you can do 'bout that now."`

> 			`	Weland opens up a palm-display with star coordinates on it. "They are part of the Core Mining Concern. Semi-sanctioned criminals, slavers, and all around bad guys. Led by the likes of the Gwesklin and other hereditary slaver families. Though they can travel through civilized space unmolested, they do like to lay low in the Naper system, especially after altercations. That's where I'd wager you'd find Mr. Gwesklin, his ship, and the stone he stole from us.`

Reviewer: ```suggestion
			`	Weland opens up a palm-display with star coordinates on it. "They are part of the Core Mining Concern. Semi-sanctioned criminals, slavers, and all around bad guys. Led by the likes of the Gwesklin and other hereditary slaver families. Though they can travel through civilized space unmolested, they do like to lay low in the <waypoints> system, especially after altercations. That's where I'd wager you'd find Mr. Gwesklin, his ship, and the stone he stole from us.`
```

			has "outfit: Tear of Skade"
	on offer
		payment 3000000
		conversation
			branch skade
				not "outfit: Blood of Surtr"
				has "outfit: Tear of Skade"
			`Weland's laboratory has mostly been cleaned up, and repaired. Weland is waiting for you at his work bench and gently places the Blood of Surtr on some sort of device.`
				goto singingstones
			label skade
			`Weland's laboratory has mostly been cleaned up, and repaired. Weland is waiting for you at his work bench and gently places the Tear of Skade on some sort of device.`
			label singingstones
			`	"It sings." Weland almost whispers, as a display shows acoustic waves emanating from within the stone. "The crystalline lattice within has been re-arranged in ways no one has ever dreamt of, in order to generate and produce waves that appear to alter the state of matter."`
			`	Weland shakes his head, "Not only is this far advanced beyond anything humans could create, or recreate, but it's an entire untapped field of study." Weland trembles with awe, "It's remarkable. Neither we, nor humans, nor any other race I know of, has deigned to interact with rock, crystal, and matter in this fashion."`
			choice

> 				`	"What do you mean, 'we'? Aren't you human?"`

Reviewer: ```suggestion
				`	"What do you mean, 'we'? Aren't you human?"`
				`	"'We'?"`
```

				goto questions
			label harmonize
			`	Weland looks at you wistfully, and then, with a tinge of sadness, shakes his head. "I wager it's a sight too dangerous for that. Each stone is grown, or sung differently, it would take me months to figure out how to harmonize another one. And in the meantime, I am still watched, still unable to leave, still the property of others. If you left somethin' that valuable with me again, they'd take it, and this time to somewhere you wouldn't find it."`
			`	He shrugs, "Perhaps, some other time. On a different world, in different circumstances. But, not now, not here, and not like this."`
				goto questions
			label offworld
			`	He gives you a patronizing smile. "Folks that did this to me. They've been doin' this kind of work for a while, and I, well, I am like the crown jewel of their collection. They have hooks into me, inside of me, all around me, that prevent me from leaving, at least alive. They like to give me the illusion of space, pretend they are... genteel 'bout the whole thing. But make no mistake, I am not free, and I cannot go."`

> 			`	His face briefly flashes with mirth, "For now, that is."`

Reviewer: ```suggestion
			`	His face briefly flashes with mirth. "For now, that is."`
```
Does this mean there will be more story with Weland later?

Author: Aye, twas the plan.

ship "Falcon" "Falcon (Gwesklin Variant)"
	outfits
		"Tear of Skade"
		"Plasma Cannon" 2
		"Torpedo Launcher" 2
		"Torpedo" 60
		"Quad Blaster Turret" 3
		"Heavy Anti-Missile Turret"

		"Breeder Reactor"
		"KP-6 Photovoltaic Array"
		"KP-6 Photovoltaic Panel" 2
		"LP144a Battery Pack"
		"D41-HY Shield Generator"
		"Large Radar Jammer"
		"Liquid Nitrogen Cooler"
		"Laser Rifle" 15

		"Impala Plasma Thruster"
		"Impala Plasma Steering"
		"Hyperdrive"

ship "Falcon" "Falcon (Corentin Variant)"
	outfits
		"Blood of Surtr"
		"Plasma Cannon" 2
		"Torpedo Launcher" 2
		"Torpedo" 60
		"Quad Blaster Turret" 3
		"Heavy Anti-Missile Turret"

		"Breeder Reactor"
		"KP-6 Photovoltaic Array"
		"KP-6 Photovoltaic Panel" 2
		"LP144a Battery Pack"
		"D41-HY Shield Generator"
		"Large Radar Jammer"
		"Liquid Nitrogen Cooler"
		"Laser Rifle" 15

		"Impala Plasma Thruster"
		"Impala Plasma Steering"
		"Hyperdrive"

mission "Hullen-Freeze revenge"
	source "Chiron"
	to offer
		has "event: killhullen"
		random < 50
	on offer
		conversation
			`As you wander through the spaceport you notice someone is following you.`
			choice
				`	(Confront them.)`
					goto confront
				`	(Try to lose them.)`

> 			`	You bob and weave through crowds, go up and down stairs, and finally duck into a vintage electronic shop. You notice the owner are selling an ancient lime green computer called an iMac. The clerk see you looking at it and explains to you it just has one working application, a game called Escape Velocity Nova. By now it's clear you've lost your tail. So you thank the clerk and return to the spaceport.`

Reviewer: ```suggestion
			`	You bob and weave through crowds, go up and down stairs, and finally duck into a vintage electronic shop. You notice the owner are selling an ancient lime-green computer called an iMac. The clerk sees you looking at it and explain that it just has one working application, a game called Escape Velocity Nova. By now it's clear you've lost your tail. So you thank the clerk and return to the spaceport.`
```

				defer
			label confront

> 			`	You turn to confront your pursuers. They quickly catch up to you. It's two men led by a tall hard-faced gray hard woman.`

Reviewer: ```suggestion
			`	You turn to confront your pursuers. They quickly catch up to you. It's two men led by a tall hard-faced woman.`
```

			`	The two men with her each grab one of your arms. "Don't worry, it'll be quick and informal."`
			`	They lead you to a small unadorned room in the spaceport. Detective Song and you enter it while the two men stand guard outside. There are two chairs in the room. Detective Song takes one and politely offers you the other.`
				goto start
			label hide
			`	"Excellent. I promise to make this brief. Now please follow me." She leads you to a small unadorned room in the spaceport. Detective Song and you enter it while the two men stand guard outside. There are two chairs in the room. Detective Song takes one and politely offers you the other.`
			label start
			choice
				`	(Take a seat.)`
				`	(Stand.)`
			`	She fishes through her pocket, pulls out a picture and hands it to you. "Do you recognize this man?"`
			`	It's Professor-Doctor Hullen-Freeze. He's standing in front of a university wing named after him.`
			choice
				`	"Yes. I met with him sometime back."`
					goto truth
				`	"No. I've never seen him in my life."`
				`	"I refuse to answer without my lawyer present."`

> 			`	"Huh. That's so interesting, because we have footage of you entering the building he works in." She pulls out a small screen that shows you, Tavish and Kormak walking into a building.`

Reviewer: ```suggestion
			`	"Huh. That's so interesting, because we have footage of you entering the building he works in." She pulls out a small screen that shows you, Tavish, and Kormak walking into a building.`
```

> 			`	"Are you sure you didn't see him inside? Perhaps you just passed him in the hall?" She looks at you with an open concerned expression.`

Reviewer: ```suggestion
			`	"Are you sure you didn't see him inside? Perhaps you just passed him in the hall?" She looks at you with an openly concerned expression.`
```

			choice
				`	"Oh, that Dr. Hullen-Freeze. Yes, I did meet with him."`
					goto truth
				`	"I enter lots of buildings. Doesn't mean I talk to everyone in them."`
				`	"I refuse to answer without my lawyer present."`
			`	"Maybe you ran into his executive secretary? A Ms. Starla Stearn?" She looks at you hopefully.`
			choice
				`	"Okay. Yes, I did meet her, and I did see Dr. Hullen-Freeze."`
					goto truth
				`	"Hmm, no. Doesn't ring a bell. Sorry I can't be more help."`
				`	"I refuse to answer without my laywer present."`
			`	"You see the reason I ask is she has submitted a signed affidavit saying she met with you, two other men, and that you forced your way past her into Dr. Hullen-Freeze's office." She gives you a helpless expression, "Come on now, Captain <last>, I can't help you if you don't help yourself."`
			choice
				`	"Okay, okay. Yes I met her and him, but I didn't force my way anywhere. She led me to his office."`
					goto truth
				`	"Sorry. She must have me confused for someone else. I just was in that building to use the restroom."`
				`	"I refuse to answer without my lawer present."`

> 			`	Detective Song rubs the bridge of her nose and sighs. "Fine. You can go. I am afraid you have not been terribly helpful. If you can think of anything that may help our investigation please let us know." She knocks on the door and one of the men opens it.`

Reviewer: She says this, but there's no way to tell them anything later.

Author: Why would you? It would only be to implicate yourself.

			`	"We're done here," she tells him, and they let you back out in the spaceport.`
				decline
			
			label truth
			`	"Ah, excellent. I really appreciate the cooperation. Could you tell me exactly what happened when you met with Ms. Starla and Dr. Hullen-Freeze?"`
			choice
				`	"Starla led us to his office where we briefly spoke and then left. Dr. Tavish was an old student of his and wanted to visit him."`
					goto liebaby
				`	"I thought I was escorting a former student of his to his office, but then Tavish and Kormak went crazy and Kormak killed him. They said they'd come after me next if I said anything."`
			`	Her notebook is out in a flash. "That's Doctor Julian Tavish, and I don't really have any records on this Kormak individual, what was his full name?"`
			choice

> 				`	"Gluniairn mac Kormakr, he's from the Deep. But I am not sure where exactly."`

Reviewer: ```suggestion
				`	"Gluniairn mac Kormakr, from the Deep. But I am not sure exactly where."`
```

				`	"Tavish just called him Kormak. I don't know anything else about him."`
			`	"And who actually carried out the murder?" She's furiously taking notes.`
			choice
				`	"Tavish ordered Kormak to do it."`
				`	"Tavish did it."`
				`	"Kormak did it."`
			action
				event "prisontavish" 200

> 			`	"Perfect." she says, "Perfect." She closes her notebook.`

Reviewer: ```suggestion
			`	"Perfect," she says. "Perfect." She closes her notebook.`
```

				`	"Nothing else happened. We went in and we left. I saw nothing suspicious."`
			`	Detective Song sighs and stands up, "Very well. You've been most helpful. If you remember anything, anything at all, please do not hesitate to contact my office here on Chiron. Especially if it relates to Dr. Julian Tavish.`
			`	"Thank you for your time, Captain <last>." She knocks on the door and one of the men opens it.`
			`	"You're free to go," she says, and they let you back out into the spaceport.`
				decline

event "prisontavish"

mission "Hullen-Freeze revenge 2"
	source
		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
	to offer
		has "event: prisontavish"
		random < 15
	on offer
		conversation
			`As you saunter through the spaceport you hear a familiar name playing on one of the video screens and look up.`
			`	"After a month long manhunt Chiron authorities have captured the notorious murderer Julian Tavish. He is charged with the savage killing of the beloved scientist and public speaker Dr. Hullen-Freeze. Dr. Hullen-Freeze had brought Julian in, despite his criminal past, in an attempt to rehabilitate the young man, and was killed for it.`

> 			`	The death of Dr. Hullen-Freeze is a tragedy not just for the Republic, but the entire human race. Stay tuned for more coverage after the break."`

Reviewer: ```suggestion
			`	"The death of Dr. Hullen-Freeze is a tragedy not just for the Republic, but the entire human race. Stay tuned for more coverage after the break."`
```

			`	"But, here you are, flying free with one on your ship, doin' whatever it is these strange stones do. I'd pay a pretty penny for somethin' like that. How about I waive the fee for fixin' up your artifact, and pay you ten million credits for that 'Blood of Surtr' stone of yours?"`

			label bloodchoice
			choice
				`	(Refuse to sell the Blood of Surtr.)`
					goto refusesurtr
				`	(Agree to sell the Blood of Surtr.)`
					goto sellsurtr
			
			label bloodsell
			`	"Now that we got that all settled. There's one other item I was wantin' to discuss. Somethin' in the ship schematics you sent over caught my eye. I think you had it labeled 'Blood of Surtr.' Now, if that is what I think it is, well, it's worth a lot of money. I never met nobody that has laid eyes on a legendary Deep Stone, let alone owned one. It's phantom tech kind of stuff you hear about from spooks and conspiracy nuts. Most say it isn't even real, and if it is, only a few have ever been found, and those the government holds under lock and key.`

> 			`	"But, here you are flying free with one on your ship doin' whatever it is these strange stones do. I'd offer to buy it off ya, but you just took most my cash.`

Author: aYE
